shell/platform/embedder/embedder.h

```diff
@@ -1051,6 +1051,9 @@ extern const int32_t kFlutterSemanticsNodeIdBatchEnd;
 /// (i.e., during PipelineOwner.flushSemantics), which happens after
 /// compositing. Updates are then pushed to embedders via the registered
 /// `FlutterUpdateSemanticsCallback`.
+///
+/// @deprecated     Use `FlutterSemanticsNode2` instead. Adding members to this
+///                 struct breaks the ABI of `FlutterUpdateSemantics`.
```

Comment:
The message here is addressing two different audiences: users of this API should use `FlutterSemanticsNode2` but authors of this API need to know not to add to it because it will break ABI compatibility.

Maybe something like:
```suggestion
/// @deprecated     Use `FlutterSemanticsNode2` instead. In order to preserve
///                 ABI compatibility for existing users, no new fields will be
///                 added to this struct. New fields will continue to be added
///                 to `FlutterSemanticsNode2`.
```

Reply:
Nice, this is definitely clearer!